src/style.css

```
@@ -1,3 +1,81 @@
 body {
+  background-color: #ddd;
+  color: black;
   margin: 0;
+  min-width: 1024px;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  --blue-color-for-link: #00acdc;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.img__link {
+  display: flex;
+  margin: 0;
+  text-decoration: none;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+:not(:first-of-type).nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  height: 60px;
+  position: relative;
+  color: black;
+}
+
+.is-active {
+  color: var(--blue-color-for-link);
+}
+
+:not(:first-of-type) .nav__link:hover {
```

why do you need this complicated selector? You can set this style for all `.nav__link`s
```suggestion
.nav__link:hover {
```

---

It was just continue of similar decission in the previous part of code. My fault. You're right. Corrected.